### ISSUE TITLE
Job posts to message boards

### ISSUE THREAD

**Issue by JuanCaicedo** [NodeDC/organizers#43]
Robbie and I were talking today about what to do with job posts to the message board on meetup. We felt that at least for NodeSchool, we would prefer not to have them. The only exception would be if a person attended the event and wanted to post a job afterwards. 

How does everyone feel about that policy?

**Comment by JuanCaicedo** [comment]
Related: I personally like the idea of having a couple minutes as we start an event for people who are hiring to announce that. Most of us aren't looking for jobs, but I think for the people who are, this would be a nice thing to have. I think @danmactough wasn't too into the idea, but I'd love to hear what the rest of you think :smiley:

**Comment by johnelliott** [comment]
@JuanCaicedo That sounds good to me, particularly for NodeSchool events. :+1:

**Comment by mcwhittemore** [comment]
I'd try to stay away from having people who are hiring talk. I've seen a few groups do the `raise your hand if your hiring, everyone who needs a job, talk to these people after the meetup in x corner of the space`. This helps keep it short, sweet and targeted at people who are interested.

**Comment by danmactough** [comment]
I agree with what @mcwhittemore said. I really just want what's best for the community and don't really know what that is. I have no objection to trying to let companies ~~shill~~ announce for a couple minutes. I do worry that it becomes hard to control. Crazy idea -- we can't go around the room and ask everyone to do the "5 words" intro when we've got 100 people, but we could ask everyone who's hiring to do it. :smile:

**Comment by JuanCaicedo** [comment]
@danmactough that's actually what I was thinking too! We can give them each like 30 seconds, I doubt there'll be more than 5 or so companies per event.

**Comment by danpaz** [comment]
Unfortunately we don't really have control over who posts what to the comments, and these generally trigger an email to the entire group (see [here](http://www.meetup.com/node-dc/events/226214228/?comment_table_id=459288260&comment_table_name=event_comment) and [here](http://www.meetup.com/node-dc/events/226957743/?comment_table_id=459288087&comment_table_name=event_comment)). I think we already encourage job announcements during the intro to the meetup, which is great. We should definitely discourage these kinds of job announcements on the message board, but I think they're rare enough anyway.

**Comment by joshfinnie** [comment]
I would also like to see a separate coordinated job board throughout all the JS meetups maybe not NodeDC specific, if [this asshole ever finishes](https://github.com/joshfinnie/jobs.nodedc.com) we might have one.

I think we can blast email the group with notification of allowing messages to the message board and direction on how to edit their email settings if this is a thing we want to do.

**Comment by JuanCaicedo** [comment]
@danpaz In this case, I deleted it and reached out to the poster explaining how we feel about it. This is what I said:

> Hello <person>
> 
> I'm writing to you about a message you posted to January's NodeSchoolDC event.  
> 
> We generally do not allow anyone to post job requirements to our message board, unless they have attended the event.  
> 
> I definitely encourage you to attend our events in the future! That's the best way to connect with interested developers.

**Comment by danmactough** [comment]
@JuanCaicedo :+1: You have a future in diplomacy.

**Comment by jonathana** [comment]
@danmactough :+1: 

The very few times this came up on my watch the ban hammer was unleashed :smile: .

More generally, do we really currently have a very big problem with recruiting pitches creeping in?  It's great to have a policy and be clear about it, but if it's the odd or occasional mis-use, @JuanCaicedo's light touch approach is probably the right one (and honestly actually the one I used).

I personally liked taking the tack that nodeDC was a group for practicing coding and engineering professionals focused on the technical side of nodejs and its ecosystem.  If it's clear somewhere, then there's a policy to refer to later.

J
